### PR TITLE
SSM access to Fargate ECS

### PR DESCRIPTION
**Pull request by igorkotof**
- Add ability to have SSM access to Fargate (ECS)

**Test**:
[asciicast](https://asciinema.org/a/P0bE82FkHtrPO7siwpJxSmqj3)

### CODE REVIEW

**Comment by igorkotof** [comment]
[Add ability to have SSM access to Fargate / ECS sidecar containers (like datadog but for SSM](https://trello.com/c/eIpDaYBn/170-add-ability-to-have-ssm-access-to-fargate-ecs-sidecar-containers-like-datadog-but-for-ssm)

File: aws/main.mk [hunk 1]
```diff
@@ -48,6 +48,10 @@ SSM_CLEANUP_ON_MAC_OS ?= rm -rf sessionmanager-bundle sessionmanager-bundle.zip
 SSM_CLEANUP_ON_LINUX_OS ?= rm -rf session-manager-plugin$(LINUX_PACKAGE_EXT)
 CMD_SSM_CLEANUP ?= $(shell echo $$(if [ "$(OS_NAME)" = "Linux" ]; then echo "$(SSM_CLEANUP_ON_LINUX_OS)"; else echo "$(SSM_CLEANUP_ON_MAC_OS)"; fi))
 
+# SSM access to Fargate ECS
+SSM_MI_TARGET ?= $(shell $(AWS) ssm describe-instance-information | $(JQ) -er '.InstanceInformationList[] | select(.Name == "$(SVC)") | .InstanceId' > tmp && cat tmp | head -1 && rm -rf tmp )
+CMD_SSM_TO_FARGATE ?= aws --profile $(AWS_PROFILE) ssm start-session --target $(SSM_MI_TARGET)
```
Comment (AutomationD) [inline aws/main.mk:53]: I'd add a comment here that we have to use local `aws` due to SSM agent.

**Review by AutomationD — APPROVED** [review]
Looks good with minor comment

File: aws/main.mk (hunk 1 above)
Comment (igorkotof) [reply aws/main.mk:53]: yes, let me add

**Pull request merged**